scripts/zkai_hybrid_proof_pressure_selector_gate.py

```
@@ -0,0 +1,695 @@
+    expect_equal(summary.get("nanozk_paper_reported_bytes"), EXPECTED_NANOZK_REPORTED_BYTES, "summary NANOZK")
+    if summary.get("attack_next_count", 0) < 1:
+        raise HybridSelectorError("selector has no attack-next route")
+    if summary.get("no_go_now_count", 0) < 1:
+        raise HybridSelectorError("selector has no no-go-now route")
+    if summary.get("go_gate") != "GO_SELECTOR_HAS_ATTACK_NEXT_AND_NO_GO_NOW_WITH_ZERO_COMPARABLE_ROWS":
+        raise HybridSelectorError("GO gate drift")
+    if not isinstance(payload.get("source_artifacts"), list) or len(payload["source_artifacts"]) != 5:
+        raise HybridSelectorError("source artifact inventory drift")
+    for descriptor in payload["source_artifacts"]:
+        if not isinstance(descriptor, dict):
+            raise HybridSelectorError("source artifact descriptor must be an object")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)